.github/workflows/3d_parallelism_unit_tests.yaml

```diff
@@ -59,6 +59,7 @@ jobs:
         --durations=0 \
         --ignore tests/kernels \
         --ignore tests/fp8 \
+        --ignore tests/test_llama.py \
```

Comment:
Can we add a flag like this https://github.com/huggingface/nanotron/blob/6a4a537054ffdd68d1d352c111cd1d93762ede95/tests/kernels/test_layer_norm.py#L8 after converting that test to a functional test, instead of running it through the command line?

Reply:
Do you mean to add  @pytest.mark.fa2 to my test? But I don't want my test to be run with the fa2 workflow.

Reply:
ah sorry, don't add `pytest.mark.fa2` beacuse that cicd run ignores fa2 flag